build/download_kubectl.ts

```diff
@@ -117,7 +117,7 @@ class KubectlDownloader {
 }
 
 const downloadVersion = packageInfo.config.bundledKubectlVersion;
-const baseDir = path.join(process.env.INIT_CWD, "binaries", "client");
+const baseDir = path.join(__dirname, "..", "binaries", "client");
```

Comment:
What's the reason behind this change?

Reply:
Because on windows `INIT_CWD` is not defined.